eiffel-vocabulary/EiffelIssueDefinedEvent.md

```diff
@@ -69,52 +66,39 @@ This section describes which link types are valid for this event type. For detai
 __Required:__ No  
 __Legal targets:__ Any  
 __Multiple allowed:__ Yes  
-__Description:__ Identifies a cause of the event occurring. SHOULD not be
-used in conjunction with __CONTEXT__: individual events providing __CAUSE__
-within a larger context gives rise to ambiguity. It is instead recommended to
-let the root event of the context declare __CAUSE__.
+__Description:__ Identifies a cause of the event occurring. SHOULD not beused in conjunction with __CONTEXT__: individual events providing __CAUSE__within a larger context gives rise to ambiguity. It is instead recommended tolet the root event of the context declare __CAUSE__.
 
 ### CONTEXT
 __Required:__ No  
-__Legal targets:__
-[EiffelActivityTriggeredEvent](../eiffel-vocabulary/EiffelActivityTriggeredEvent.md),
-[EiffelTestSuiteStartedEvent](../eiffel-vocabulary/EiffelTestSuiteStartedEvent.md)  
+__Legal targets:__ [EiffelActivityTriggeredEvent](../eiffel-vocabulary/EiffelActivityTriggeredEvent.md), [EiffelTestSuiteStartedEvent](../eiffel-vocabulary/EiffelTestSuiteStartedEvent.md)  
 __Multiple allowed:__ No  
-__Description:__ Identifies the activity or test suite of which this event
-constitutes a part.
+__Description:__ Identifies the activity or test suite of which this eventconstitutes a part.
 
 ### FLOW_CONTEXT
 __Required:__ No  
-__Legal targets:__
-[EiffelFlowContextDefinedEvent](./EiffelFlowContextDefinedEvent.md)  
+__Legal targets:__ [EiffelFlowContextDefinedEvent](../eiffel-vocabulary/EiffelFlowContextDefinedEvent.md)  
 __Multiple allowed:__ No  
-__Description:__ Identifies the flow context of the event: which is the
-continuous integration and delivery flow in which this occurred – e.g. which
-product, project, track or version this is applicable to.
+__Description:__ Identifies the flow context of the event: which is thecontinuous integration and delivery flow in which this occurred – e.g. whichproduct, project, track or version this is applicable to.
```

Comment:
```suggestion
__Description:__ Identifies the flow context of the event: which is the continuous integration and delivery flow in which this occurred – e.g. which product, project, track or version this is applicable to.
```

Reply:
Fixed in 837dc1f55476b3ca0ba2d9e8c235f39c4cd7ecf3.